ios/PhyWeb/UI/PWBeaconsViewController.m

```diff
@@ -329,6 +329,21 @@ - (void)_updateBeaconsNow {
 - (void)_sort {
   BOOL stableMode = [[PWBeaconManager sharedManager] isStableMode];
 
+  BOOL hasRank = NO;
+  for(PWBeacon * beacon in _beacons) {
+    if ([beacon hasRank]) {
+      hasRank = YES;
```

Comment:
We can start assuming that PWS will return a rank.
And if beacon have no rank assign them 1000.
I'll change this pull request.